src/if/os_interface_bridge.cpp

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (c) 2016 Dell Inc.
```

Comment:
2016 -> 2018

Reply:
ignoring.. i got a script that will take care of copyright info, but will do it in next commit (rc0)